docs/esterni/analisi_dei_requisiti/analisi_dei_requisiti.tex

```diff
@@ -530,14 +530,16 @@ \subsection{Requisiti funzionali}
 \hline
 RFO50 & La landing page deve tracciare l'id dell'oggetto venduto tramite metodo GET. & Obbligatorio & \underline{\href{https://www.math.unipd.it/~tullio/IS-1/2021/Progetto/C2.pdf}{Capitolato}}, \underline{\href{https://github.com/iota97/WinningSoftwareSolution/blob/main/public/esterni/verbali/2021_12_22_E.pdf}{verbale del 22/12/2021}}, \underline{\href{https://github.com/iota97/WinningSoftwareSolution/blob/main/public/interni/verbali/2021_12_23_I.pdf}{verbale del 23/12/2021}} \\
 \hline
-RFO13 & L'acquirente con wallet connesso deve vedere il prezzo da pagare in dollari nella landing page. & Obbligatorio & UC7 \\
+RFOX & L'acquirente con wallet connesso deve vedere il prezzo dell'oggetto nella landing page. & Obbligatorio & UC7.1 \\
 \hline
-RFO14 & L'acquirente con wallet connesso poter confermare il pagamento nella landing page. & Obbligatorio & UC8 \\
+RFOX & L'acquirente con wallet connesso deve vedere il wallet del venditore dell'oggetto nella landing page. & Obbligatorio & UC7.2 \\
 \hline
-RFO15 & Il sistema deve mostrare un messaggio "ordine annullato" se l'acquirente con wallet connesso annulla il pagamento sulla landing page. & Obbligatorio & UC8.1 \\
+RFO14 & L'acquirente con wallet connesso poter confermare il pagamento nella landing page. & Obbligatorio & UC8 \\
 \hline
 RFO16 & Il sistema deve cancellare la sessione di pagamento se l'acquirente con wallet connesso annulla il pagamento sulla landing page. & Obbligatorio & UC8.1 \\
 \hline
+RFO15 & Il sistema deve reindirizzare l'acquirente sulla home page di \textit{google.com} se l'acquirente con wallet connesso annulla il pagamento sulla landing page. & Obbligatorio & UC8.1 \\
```

Comment:
Vedi sopra. Il link per pagare un oggetto sarà sito.com/land?id=ID&r=ecommerce.com. Una volta effettuato il pagamento, ShopChain reindirizzerà l'utente su ecommerce.com?id=id_transazione_pagata in modo tale che l'ecommerce possa salvarsi l'id del pagamento.